src/utils/__tests__/currency.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { numberToCurrencyText } from '../currency';
+
+describe('currency', () => {
+    describe('numberToCurrencyText', () => {
+        it('should convert a number to US-supported currency format', () => {
+            expect(numberToCurrencyText(10000)).toBe('10,000.00');
+        });
+    });
+});
```

Comment:
can you please check if we can replace this with `formatMoney` from `deriv-utils`?